components/benefit_sponsors/spec/controllers/benefit_sponsors/profiles/registrations_controller_spec.rb

```diff
@@ -345,6 +345,15 @@ module BenefitSponsors # rubocop:disable Metrics/ModuleLength
             it "should redirect to new for general agency" do
               expect(response.location.include?("new?profile_type=general_agency")).to eq true if profile_type == "general_agency"
             end
+
+            it "should create stuff person with no ssn" do
```

Comment:
@hKalesnikavaideacrew this looks good. Minor typo; can you change "stuff" to "staff" same with the PR title.